frontend/app/components/nav-bar.js

```diff
@@ -66,4 +70,22 @@ export default class NavBarComponent extends Component {
       }
     }, 800);
   }
+
+  @action
+  copyCcliLogin() {
+    this.fetchService
+      .send(`/api/api_users/ccli_token`, { method: "GET" })
+      .then((response) => {
+        response.json().then((json) => {
+          this.clipboardService.copy(
+            `cry login ${btoa(`${json.username}:${json.token}`)}@${
+              window.location.origin
```

Comment:
I am wondering whether it would make more sense to create the whole string in the backend and use backend ENV variables to do this :thinking: 
`ENV['RAILS_HOST_NAME'] ` would probably fit best. (I checked our Openshift environment and it is correctly set to the relevant url.)
I'll let @mtnstar chime in here :grin:

Reply:
I would leave it as it is for the moment. 
not so sure if it's a good idea to rely on RAILS_HOST_NAME env